FalmouthFalcons/iteration-3
# Ticket 9 mostly working, deleting with incorrect redirect

Description:
Ticket 9 mostly working, deleting product with incorrect redirect
-9. User can remove a product listing #22

Number of Fixes:
-1

Related Ticket(s):
Ticket 9 

Problem to Solve:
deleting product with incorrect redirect

Proposed Changes:
deleting product and redirect to products

Expected Behavior:
deleting product with incorrect redirect

Steps to Test Solution
1) git fetch
2) git checkout ticket/jq
3) bundle install, rails db:migrate 

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
